### DISCUSSION TITLE
It would be useful to be able to name iterations

### DISCUSSION THREAD

**Discussion by akumria** [community/community#12625] [Projects and Issues]
For example, adopting this method means that if, externally, other groups are on iteration 8 - we will not be on the same number.

Another issue is that teams sometimes name them with FY2022, etc.

**Comment by milanvanschaik** [comment]
@akumria Is what you’re trying to achieve to have a custom name per iteration? This seems to be possible already. Go to project’s settings > Name of your iteration field > Click on title of the iteration to edit it.
